mlrun/feature_store/feature_set.py

```diff
@@ -366,6 +368,7 @@ def add_aggregation(
         :param state_name: optional, graph state name
         :param after:      optional, after which graph state it runs
         :param before:     optional, comes before graph state
+        :param emit_policy optional. Define emit policy of the aggregations
```

Comment:
"Define emit policy of the aggregations" doesn't really help understanding what it is, maybe add an example of an example policy or something